ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -151,6 +151,9 @@ title: Ansible Collection Role eos_validate_state - Preview Integration with ANT
 - (New) AvdTestAPIHttpsSSL (No Ansible tags, use the new `skipped_tests` variable instead)
   - VerifyAPIHttpsSSL: Validate eAPI HTTPS SSL profile status.
 
+- (New) AvdTestStun (No Ansible tags, use the new `skipped_tests` variable instead)
+  - AvdTestStun: Validate STUN clients source address and port.
```

Comment:
Can we add more details about the test here? Since it's related to WAN devices, we should mention it somehow.